pkg/commands/git_commands/remote_loader.go

```diff
@@ -84,8 +84,16 @@ func (self *RemoteLoader) GetRemotes() ([]*models.Remote, error) {
 func (self *RemoteLoader) getRemoteBranchesByRemoteName() (map[string][]*models.RemoteBranch, error) {
 	remoteBranchesByRemoteName := make(map[string][]*models.RemoteBranch)
 
+	sortOrder := "refname"
+	switch strings.ToLower(self.AppState.RemoteBranchSortOrder) {
+	case "alphabetical":
+		sortOrder = "refname"
```

Comment:
This assignment is a no-op, and this (slightly) bugs me. On the other hand, the code documents what the official enum values are, which is nice. Not sure what to do about it; I experimented with other versions of expressing this code, including this one:
```go
	var sortOrder string
	switch strings.ToLower(self.AppState.RemoteBranchSortOrder) {
	case "date":
		sortOrder = "-committerdate"
	default:
		// including the "alphabetical" case
		sortOrder = "refname"
	}
```
but I'm not sure that's better. Your call.

Reply:
I've also written out the `default` branch once but stuck with the version that spells out valid sort order strings. How about `fallthrough`? (apparently the position of `default` is not important, other cases are checked first)
```golang
	var sortOrder string
	switch strings.ToLower(self.AppState.RemoteBranchSortOrder) {
	default:
		fallthrough
	case "alphabetical":
		sortOrder = "refname"
	case "date":
		sortOrder = "-committerdate"
	}

```

Reply:
I find this a little bit brittle; if we add more sort options later, we have to be careful not to add them *before* alphabetical, otherwise we change the default.

I also considered fallthrough, but would have done it the other way round ("alphabetical" falling through to default at the end). That's also redundant though.

It's such a small thing that it's probably not worth talking about so long. Maybe the best version is
```go
	var sortOrder string
	switch strings.ToLower(self.AppState.RemoteBranchSortOrder) {
	case "alphabetical":
		sortOrder = "refname"
	case "date":
		sortOrder = "-committerdate"
	default:
		sortOrder = "refname"
	}
```
It's almost like your initial version, except that it never assigns a value when the variable already has that value. (shrug)

Reply:
Updated as suggested in https://github.com/jesseduffield/lazygit/pull/3171/commits/3c710f5f04c1d8a8f752bccd66969b9e4c228c7d